db/schema.rb

```diff
@@ -57,24 +57,6 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "user_followers", force: :cascade do |t|
```

Comment:
Removed because there is no use case for this anymore. This was my initial idea but then ended up going with 'friendships'